renovate.json

```
@@ -16,12 +16,10 @@
       ],
       "matchDatasources": ["docker"],
       "matchPackageNames": [
-        "registry.access.redhat.com/ubi9/ubi-minimal",
-        "registry.access.redhat.com/ubi9/ubi"
+        "registry.access.redhat.com/ubi9/ubi-minimal"
       ],
       "enabled": true,
       "versioning": "redhat",
```

```suggestion
      "versioning": "docker",
```

I think that if we intend to use the ubi-minimal with the latest tag and the sha, the `versioning` should be changed also, otherwise I think we would not get the image bump PRs.

From Renovate docs on versioning ([link](https://docs.renovatebot.com/modules/versioning/redhat/)):

> Red Hat version definitions follow this pattern:
>
> - the version of the main component, where major is required, but minor and patch are optional
> - optionally a hyphen followed by release information
> - the version of Red Hat's release as an integer, optionally followed by a timestamp like: 1645808164
> - optionally a "v"-prefix
> 
> Examples of valid Red Hat versions:
> 
> - 1
> - 8.5
> - 7.9-628
> - v0.4.0-383
> - 9.0.0-19.1655192132

`docker` versioning should be suitable here I think ([docs link](https://docs.renovatebot.com/modules/versioning/docker/)).

Alternatively we could stick to the semantic versioning tag for the ubi-minimal image and keep this config unchanged.

---

Good point, I asked this question in the FAQ doc (will give you a link in Slack).

---

I tested with renovate dry-run and it looks to be ok with the `redhat` versioning and the `latest@sha256`.

I used an older sha in the Containerfile just to see if the bump would be detected, but kept the `redhat` versioning in the `renovate.config`:

```yaml
         {
           "branchName": "renovate/registry.access.redhat.com-ubi9-ubi-minimal-latest",
           "prNo": null,
           "prTitle": "UPSTREAM: <carry>: Update registry.access.redhat.com/ubi9/ubi-minimal:latest Docker digest to 61d5ad4",
           "result": "not-scheduled",
           "upgrades": [
             {
               "datasource": "docker",
               "depName": "registry.access.redhat.com/ubi9/ubi-minimal",
               "displayPending": "",
               "currentValue": "latest",
               "currentDigest": "sha256:53ea1f6d835898acda5becdb3f8b1292038a480384bbcf994fc0bcf1f7e8eaf7",
               "newValue": "latest",
               "newDigest": "sha256:61d5ad475048c2e655cd46d0a55dfeaec182cc3faa6348cb85989a7c9e196483",
               "packageFile": "Containerfile.externaldns",
               "updateType": "digest",
               "packageName": "registry.access.redhat.com/ubi9/ubi-minimal"
             }
           ]
         }

```

So I take my comment/suggestion back.